src/app/globals.css

```
@@ -1,5 +1,5 @@
-@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css");
-@import "tailwindcss";
+@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css');
```

Stylelint (17.11.0)

[error] Expected "url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css')" to be "'https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css'" (import-notation)

(import-notation)